### PR TITLE
adding fine overlay connectivity check

### PR DESCRIPTION
**Pull request by ladoramkershner**
This is where I ended last week on the cnet health project. I updated the code and notebook to include a more fine overlap connectivity check. There is a TO DO note in the added function and I think the notebook documentation has more to be desired.

### CODE REVIEW

File: autocnet/graph/network.py [hunk 1]
```diff
@@ -2295,12 +2283,67 @@ def connected_overlays(self, filters={'ignore': False}, size_threshold=0):
...
+            # create graph edges
+            for pid, g in df.groupby('pointid'):
+                edge_pool = np.sort([row['imageid'] for i, row in g.iterrows()])
+                graph.add_edges_from(list(combinations(edge_pool, 2)))
+
+            # evaluate connectivity of overlap
+            fully_connected_number_of_edges = scipy.special.comb(graph.number_of_nodes(),2)
+            all_edges = list(combinations(graph.nodes, 2))
+            if graph.number_of_edges() == fully_connected_number_of_edges:
+                print(f'Overlap {oid} is fully connected')
```
Comment (jessemapel) [inline autocnet/graph/network.py:2341]: This function should not print. It's caller can decide if it wants to print or not, but this should just do its processing.

**Review by jessemapel — COMMENTED** [review]
I'll keep working on this PR to get some things ironed out, but this is fully functional. 👍

**Comment by ladoramkershner** [comment]
We should probably add a point threshold

**Comment by jessemapel** [comment]
Linux py 3.6 environment solve crapped out, this is good to merge.

**Review by jessemapel — APPROVED** [review]

**Pull request merged**